rpa-builder/6.0/modules/ROOT/pages/workbench-toolbar-overview.adoc

```diff
@@ -0,0 +1,213 @@
+= Workbench Toolbar Overview
+
+Next to the the Workbench tab bar is the Workbench toolbar, which enables you to perform different actions related to the Workflow and its elements.
+
+image::workbench-toolbar.png[The Workbench Toolbar, 75%, 75%]
+
+Some of these actions apply to the selected objects in the Workflow. The unavailable functions are grayed out depending on which Workflow object is selected.
+
+The following is a list of the available buttons in the Workbench Toolbar and their corresponding action:
+
+* (image:copy-icon.png[The copy icon, 2%, 2%]) <<copy-paste, Copy selected object(s)>>
+* (image:paste-icon.png[The paste icon, 2%, 2%]) <<copy-paste, Paste copied object(s)>>
+* (image:arrow-rotate-left-icon.png[The Undo icon, 2%, 2%]) <<undo-redo, Undo previous action>>
+* (image:arrow-rotate-right-icon.png[The Redo icon, 2%, 2%]) <<undo-redo, Redo action>>
+* (image:arrow-up.png[The Move selected object up icon, 2%, 2%]) <<move-up-down, Move selected object up>>
+* (image:arrow-down.png[The Move selected object down icon, 2%, 2%]) <<move-up-down, Move selected object down>>
+* (image:delete-icon-alt.png[The Delete selected object icon, 2%, 2%]) <<delete, Delete selected object(s)>>
+* (image:run-wizard-icon.png[The Run wizard for selected object icon, 2%, 2%]) <<start-wizard, Run wizard for selected object>>
+* (image:quick-creation-icon.png[The Start Quick Creation Mode icon, 2%, 2%]) <<start-quick-creation, Start Quick Creation Mode>>
+* (image:validate-icon.png[The Validate Workflow icon, 2%, 2%]) <<validate, Validate Workflow>>
+* (image:play-icon.png[The Run Workflow icon, 2%, 2%]) <<run-workflow, Run Workflow>>
+* (image:debug-icon.png[The Debug Workflow icon, 2%, 2%]) <<debug-workflow, Debug Workflow>>
+* (image:print-icon.png[The Print icon, 2%, 2%]) <<print-workflow, Print>>
+* (image:wrench-icon.png[The Tools icon, 2%, 2%]) <<tools, Tools>>
+
+In addition to the Workbench Toolbar, you can find the <<search, Search>> bar at the end of the Workbench:
+
+image:search-bar.png[The Search bar, 40%, 40%]
+
+[[copy-paste]]
+== Copy and Paste Workflow Object(s)
+Pressing *Copy* (image:copy-icon.png[The copy icon, 2%, 2%]) copies the selected Workflow object(s) and all its underlying elements into the clipboard. Select the target object and press *Paste* (image:paste-icon.png[The paste icon, 2%, 2%]) to paste it. You can also use _Ctrl + C_ for copying and _Ctrl + V_ for pasting.
```

Comment:
```suggestion
Clicking *Copy* (image:copy-icon.png[The copy icon, 2%, 2%]) copies the selected Workflow objects and their underlying elements to the clipboard. Select the target object and click *Paste* (image:paste-icon.png[The paste icon, 2%, 2%]) to paste it. You can also use _Ctrl + C_ for copying and _Ctrl + V_ for pasting.
```

Reply:
We don't use (s). Using s by itself covers single and plural. Here is the blurb from the style guide:

"(s)"
Don't add (s) or (es) to a singular noun to indicate that it can be singular or plural.

In general, use plural, but be guided by meaning:

Before
To add row(s) or column(s) to a table….

After 
To add rows or columns to a table....

If it's important to indicate both, use "one or more."